README.md

```diff
@@ -249,6 +249,7 @@ View more style guides at [Website Style Guide Resources](http://styleguides.io/
 * [react-with-styles](https://github.com/airbnb/react-with-styles)
 * [styled-jsx](https://github.com/zeit/styled-jsx)
 * [styled-components](https://github.com/styled-components/styled-components)
+* [stylin](https://github.com/sultan99)
```

Comment:
The provided link is not referring to stylin. Please update the link.

Reply:
Oh, epic fail

Reply:
@Rishabh04-02  it is fixed. Thank you!